.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Consider fixing this `no new line at end of file` error

Reply:
@Nziranziza Thank you! I fixed that